src/node-file-trace.ts

```diff
@@ -22,6 +22,8 @@ export async function nodeFileTrace(files: string[], opts: NodeFileTraceOptions
     job.stat = opts.stat;
   if (opts.readlink)
     job.readlink = opts.readlink;
+  if (opts.resolve)
+    job.resolve = opts.resolve;
```

Comment:
I hate to be too pedantic, but can we implement the model here as `job.resolve = opts.resolve || defaultResolve`, where `job.resolve` is the resolve function that is called in the first place?

Reply:
implemented that suggestion 👍